packages/components/tooltip/src/tooltip.js

```diff
@@ -321,84 +342,9 @@ Tooltip.propTypes = {
     WrapperComponent: PropTypes.elementType,
   }),
   /**
-   * Provides a way to fine-tune an appearance of underlying Popper tooltip element. For more information, please check [Popper.js documentation](https://popper.js.org/popper-documentation.html#modifiers).
+   * Provides a way to fine-tune an appearance of underlying Popper tooltip element. For more information, please check [Popper.js documentation](https://popper.js.org/docs/v2/modifiers/).
    */
-  modifiers: PropTypes.shape({
```

Comment:
The breaking changes in propper didn't include these right? I remember @pa3 added them back then to use them. Knowing that we bind all of UIKit to a breaking change if popper changes anything here potentially.

Reply:
I've checked and at least the modifiers that he needed still work the same as before, but a few of the previously supported ones do not exist anymore in v2 😕 

So now, I guess it is considered a breaking change, because we had exposed all the modifiers in the API 😬

As for the propTypes, I just removed their explicit shape because I think we don't need to fill the props table of our Readme with the same things that already exists in Popper's docs.